src/Controller/ArticlesController.php

```diff
@@ -851,13 +851,14 @@ private function articlePageArguments(Request $request, string $id, int $version
 
                 $infoBars = [];
 
-                $latestVersion = $history->getVersions()[count($history->getVersions()) - 1]->getVersion();
+                $latest = $history->getVersions()[count($history->getVersions()) - 1];
+                $latestVersion = $latest->getVersion();
 
                 if ($item->getVersion() < $latestVersion) {
                     $infoBars[] = new InfoBar('Read the <a href="'.$this->generatePath($history).'">most recent version of this article</a>.', InfoBar::TYPE_MULTIPLE_VERSIONS);
                 }
 
-                if ($item instanceof ArticlePoA) {
+                if ($latest instanceof ArticlePoA) {
```

Comment:
If `$latest` is PoA then we can assume that all versions of the article of PoA. Looks good to me!